src/public/index.html

```diff
@@ -60,6 +60,18 @@ <h1 id="rpsStat">-</h1>
       <canvas id="rpsChart" width="200" height="100"></canvas>
     </div>
   </div>
+  <div class="container">
+    <div class="stats-column">
+      <h5>Status Codes</h5>
+      <h6 style="color: #75D701">2xx</h6>
+      <h6 style="color: #47b8e0">3xx</h6>
+      <h6 style="color: #ffc952">4xx</h6>
+      <h6 style="color: #E53A40">5xx</h6>
+    </div>
+    <div class="chart-container">
+      <canvas id="statusCodesChart" width="200" height="100"></canvas>
+    </div>
+  </div>
   <div class="footer">
```

Comment:
minor: also the footer has got `position: fixed;` at the css level and I would get rid of it as it overlaps w/ the last monitor on small screens (e.g. my MBP)